### PR TITLE
Added sync-asset-attributes script for our partner to use

### PR DESCRIPTION
**Pull request by EvelinaBuiciag**
Adde sync-asset-attributes script for our partners to use in order to sync their assets metadata DP-162 resulted from DP-121

### CODE REVIEW

File: src/assets/sync-asset-attributes.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    web3: null,
+    env: env
+  };
+  const mClient = new MyriaClient(iClient);
+  const moduleFactory = new ModuleFactory(mClient);
+  const assetManager = moduleFactory.getAssetOnchainManager();
+
+  let syncAssets;
+  try {
+    console.log(`Resfresh asset metadata...`);
```
Comment (thisonedev) [inline src/assets/sync-asset-attributes.ts]: Typo: `Resfresh` -> `Refresh`

File: README.md [hunk 2]
```diff
@@ -298,4 +298,16 @@ Requirements:
 
 ```bash
 npm run get-asset-vault-details
+```
+
+#### 5.4 Get asset vault details
```
Comment (thisonedev) [inline README.md]: I think you forgot to change the header.

`Get asset vault details` -> `Sync asset attributes`

File: package.json [hunk 3]
```diff
@@ -23,7 +23,8 @@
     "get-asset-by-id": "ts-node -r dotenv/config -r tsconfig-paths/register ./src/assets/asset-by-id.ts",
     "get-assets-by-stark-key": "ts-node -r dotenv/config -r tsconfig-paths/register ./src/assets/assets-by-stark-key.ts",
     "get-assets-by-public-key": "ts-node -r dotenv/config -r tsconfig-paths/register ./src/assets/assets-by-public-key.ts",
-    "get-asset-vault-details": "ts-node -r dotenv/config -r tsconfig-paths/register ./src/assets/asset-vault-details.ts"
+    "get-asset-vault-details": "ts-node -r dotenv/config -r tsconfig-paths/register ./src/assets/asset-vault-details.ts",
+    "sync-asset-attributes": "ts-node -r dotenv/config -r tsconfig-paths/register ./src/assets/sync-asset-attributes.ts"
```
Comment (thisonedev) [inline package.json]: Should we call this method `Sync asset metadata` or `Refresh asset metadata` so that it matches the SDK function?

**Review by thisonedev — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (EvelinaBuiciag) [reply package.json]: changed it to refresh-asset-metadata

File: README.md (hunk 2 above)
Comment (EvelinaBuiciag) [reply README.md]: Yup. Changes made

File: src/assets/sync-asset-attributes.ts (hunk 1 above)
Comment (EvelinaBuiciag) [reply src/assets/sync-asset-attributes.ts]: Typo fixed

**Pull request merged**